dell/src/main/java/org/apache/iceberg/dell/EcsClientProperties.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.dell;
+
+/**
+ * Property constants of catalog
+ */
+public interface EcsClientProperties {
+
+  /**
+   * Access key id
+   */
+  String ACCESS_KEY_ID = "ecs.s3.access.key.id";
```

Comment:
The `ecs.s3` prefix is to declare that use the S3-compatible protocol at Dell EMC ECS.
The Dell EMC ECS supports other protocols. So must add to S3 to avoid misunderstand.

Reply:
Does it matter for the users? From their perspective they are just using the ECSFIleIO, S3 is just implementation detail.

Or do you mean only s3 uses access key ID and secret access key, but you will support more credential configurations of other cloud providers, like `ecs.gcp.project-id`?

Reply:
Partially yes.
This credential configuration is for S3 and Dell EMC Atmos. The latter is rarely used recently. So named `s3` is well for our customer.
And the other protocols use others credential configurations.

See http://doc.isilon.com/ECS/3.6/API/index.html , we always specify S3 in our document.

Reply:
I see, thanks for the explanation